x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/enrollment/EnrollmentTokenGenerator.java

```diff
@@ -199,14 +201,38 @@ protected String getCaFingerprint() throws Exception {
     }
 
     static List<String> getFilteredAddresses(List<String> addresses) throws Exception {
-        List<String> filtered_addresses = new ArrayList<>();
+        List<String> filteredAddresses = new ArrayList<>();
         for (String bound_address : addresses){
```

Comment:
Just as we are changing the names according to the convention, can we also rename it to `boundAddress`?